src/sdk/documents/get-document.ts

```diff
@@ -0,0 +1,23 @@
+import { SupabaseClient } from "@supabase/supabase-js";
```

Comment:
That's actually a duplicate (almost) to a change I've made meanwhile. Only that I called it `document/get-document.ts` (rather than `documents/get-document`) :-D 

I can resolve the duplication after the merge. I guess we just need to decide on whether we want to go with plural or singular (document or documents). The current naming isn't actually consistent... Plural might be better. (In this case though, we should also include into the query `collection_id` and `collection_metadata` because that's also defined in the `Document` type.)